kubernetes/linux/main.sh

```diff
@@ -750,7 +751,11 @@ if [ "${CONTAINER_TYPE}" == "PrometheusSidecar" ]; then
 else
       echo "starting mdsd in main container..."
       # add -T 0xFFFF for full traces
-      mdsd ${MDSD_AAD_MSI_AUTH_ARGS} -e ${MDSD_LOG}/mdsd.err -w ${MDSD_LOG}/mdsd.warn -o ${MDSD_LOG}/mdsd.info -q ${MDSD_LOG}/mdsd.qos 2>>/dev/null &
+      export MDSD_ROLE_PREFIX=/var/run/mdsd-ci/default
```

Comment:
since the MDSD_ROLE_PREFIX specified, why do we need  other one in envmdsd?

Reply:
I am doing the same steps as is done for the prometheussidecar case.